pkg/config/node.go

```diff
@@ -235,11 +235,18 @@ func GetIngressConfig(kubeconfigPath string) (ingressConfig IngressConfig, err e
 	}
 
 	for _, node := range nodes.Items {
+		nodeIp := ""
 		for _, address := range node.Status.Addresses {
 			if address.Type == v1.NodeInternalIP {
-				ingressConfig.Peers = append(ingressConfig.Peers, address.Address)
+				nodeIp = address.Address
```

Comment:
I think we also need to check that we pick the IP version matching the VIP. I'm not sure selecting an IPv6 internal IP for an IPv4 VIP will work.

Even if it does, it will still trigger the warning message in the keepalived logs which means we'll keep getting bugs opened about this. :-)

Reply:
Well, I also thought picking the IP matching the VIP but it seems that picking the first IP for each node works for LB details [1] for dual-stack case.

To be on the safe side, I changed the code to pick IP according to VIP


[1]
https://github.com/openshift/baremetal-runtimecfg/blob/master/pkg/config/node.go#L369-#L382